src/lib/djerba/util/html.py

```diff
@@ -59,17 +59,22 @@ def markdown_to_html(markdown_string):
         return markdown(markdown_string)
     
     @staticmethod
-    def section_cells_begin(section_title, is_main):
+    def section_cells_begin(section_title, is_main, is_top=False):
         # begin a cell structure with title in left-hand cell, body in right-hand cell
         if is_main:
             class_suffix = 'main'
             rank = '2'
         else:
             class_suffix = 'supp'
             rank = '3'
-        template = '<hr class="big-white-line" ><div class="twocell{0}">'+\
-            '<div class="oneoftwocell{0}"><h{1}>{2}</h{1}></div>'+\
-            '<div class="twooftwocell{0}" ><hr class="big-line" >'
+        if is_top:
```

Comment:
Copy-pasting strings makes it hard to tell how they differ.
Better to have a base template variable, and in the `else` clause do something like `template=base_template+'<hr class="big-line">'`